tests/backends/test_Windows.py

```diff
@@ -10,56 +8,96 @@
     not keyring.backends.Windows.WinVaultKeyring.viable, reason="Needs Windows"
 )
 class TestWinVaultKeyring(BackendBasicTests):
-    def tearDown(self):
-        # clean up any credentials created
-        for cred in self.credentials_created:
-            try:
-                self.keyring.delete_password(*cred)
-            except Exception as e:
-                print(e, file=sys.stderr)
```

Comment:
I don't understand what's going on here. This functionality seems to have been removed without justification. This code is there for a reason, preventing the pollution of a user's own keychain when running tests. Why was it removed?

Reply:
When I ran my tests, I noticed that this function wasn't getting called.  Furthermore, the `cleanup` function in the underlying `BackendBasicTests` _was_ getting called (seemingly accomplishing the same goal, although without the `try...except`.

Looking at the `pytest` documentation, I see that there is an optional `teardown` (lowercase `"d"`) method that a test class can provide.  My guess is that this function was intended to be named `teardown` and was mistakenly named `tearDown`, which made it invisible to `pytest`.

Since the base class `BackendBasicTests` `cleanup` seemed to clean up all the unit test password in my run, I don't know that we need this one. 

I should have documented why I yanked it -- sorry about that!